src/Aquifer.API/Endpoints/Users/GetSelf/Response.cs

```diff
@@ -5,8 +5,9 @@ public class Response
     public int Id { get; set; }
     public required string Name { get; set; }
     public required IEnumerable<string> Permissions { get; set; }
-
     public required CompanyResponse Company { get; set; }
+    public int? LanguageId { get; set; }
+    public bool? HasAssignedContent { get; set; }
```

Comment:
I think this will be confusing without context, and it's not really correct when talking about a non-community user. Most users will have content assigned to them, but this will be null. I'd also argue that null isn't really a valid state for this, since a user either does or doesn't have content assigned. So I would rename this to something like `bool CanBeAssignedContent` that should be true unless it's a community user with something already assigned.

Reply:
This caused changed on the front end. So, there's a new PR for it there as well.